format/FlightSql.proto

```diff
@@ -90,6 +90,55 @@ enum SqlInfo {
    */
   FLIGHT_SQL_SERVER_READ_ONLY = 3;
 
+  /*
+   * Retrieves a boolean value indicating whether the Flight SQL Server supports executing
+   * Substrait plans.
+   */
+  FLIGHT_SQL_SERVER_SUBSTRAIT = 4;
```

Comment:
Should there also be a `FLIGHT_SQL_SERVER_SQL` indicating whether the server supports executing SQL requests, or is that mandatory?

Reply:
Hmm, I think that makes sense, though current clients are going to assume SQL support.